packages/common-all/src/types/typesv2.ts

```diff
@@ -163,6 +163,16 @@ export type NoteQuickInput = NoteProps & {
   alwaysShow?: boolean;
 };
 
+/**
+ * A reduced version of NoteQuickInput that only keeps the props necessary for
+ * lookup quick pick items
+ */
+export type NoteQuickInputV2 = Pick<NoteProps, "fname" | "vault" | "schema"> & {
```

Comment:
can we label these types as `WebExt` version so that its easy to pick out what is for web extension and what is not?

Reply:
Hm - for this even tho this is the version for web extension now, I think it should eventually be the version for everything though.

Reply:
Offline follow up: let's do a validation on dendron.yml on startup to make sure they're relative.